python/utils.py

```diff
@@ -31,9 +31,11 @@ def load_all_yaml_files() -> Dict[str, Dict[str, Dict]]:
         # Get list of all transform of certain type
         transform_names = [x.name for x in transform_type_dir_path.rglob("*/**")]
         for transform_name in transform_names:
-            transform_dir = transform_type_dir_path / transform_name
-            transform_yaml_path = transform_dir / f"{transform_name}.yaml"
+            transform_yaml_path = transform_type_dir_path / transform_name / f"{transform_name}.yaml"
+            transform_yaml_override_path = transform_type_dir_path / transform_name / constants.RASGO_DATAWAREHOUSE / f"{transform_name}.yaml"
 
+            if Path(transform_yaml_override_path).exists():
```

Comment:
`transform_yaml_override_path` is already a path obj I'm pretty sure, so no need to wrap it in a path obj again